actors/abi/bigint/bigint.go

```diff
@@ -0,0 +1,250 @@
+package big
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"math/big"
+
+	cbor "github.com/ipfs/go-ipld-cbor"
+	"github.com/polydawn/refmt/obj/atlas"
+
+	cbg "github.com/whyrusleeping/cbor-gen"
+	"golang.org/x/xerrors"
+)
+
+const BigIntMaxSerializedLen = 128 // is this big enough? or too big?
+
+func init() {
+	cbor.RegisterCborType(atlas.BuildEntry(Int{}).Transform().
+		TransformMarshal(atlas.MakeMarshalTransformFunc(
+			func(i Int) ([]byte, error) {
+				return i.cborBytes(), nil
+			})).
+		TransformUnmarshal(atlas.MakeUnmarshalTransformFunc(
+			func(x []byte) (Int, error) {
+				return fromCborBytes(x)
+			})).
+		Complete())
+}
```

Comment:
Let's remove the altas entries (and hence the dependency on refmt), but export the cborBytes methods so that an application linking this library can write their own if needed.